src/Univapay/Resources/PaymentData/TokenThreeDS.php

```diff
@@ -8,34 +8,35 @@
 use Univapay\Utility\FunctionalUtils;
 use Univapay\Utility\Json\JsonSchema;
 
-class ThreeDS
+class TokenThreeDS
 {
     use Jsonable;
 
     public $enabled;
     public $status;
     public $redirectEndpoint;
     public $redirectId;
+    public $error;
 
     public function __construct(
         $enabled = null,
         $status = null,
```

Comment:
I think we should move read only params after all write only params. Otherwise it people will always have to write `null` for this param

Reply:
Also specify class type.